backend/db/utils.py

```diff
@@ -809,17 +809,36 @@ def get_idle_connections(interval: str = '1 week'):
     return result
 
 
+def kill_idle_cons(threshold_minutes=10):
```

Comment:
## The function
Kills all older than 10 minutes and only having to do with TermHub.